amd/src/chosen_jquery.js

```diff
@@ -0,0 +1,1241 @@
+define(["jquery"], function(jQuery) {
```

Comment:
What does this library do and do we still need this? Would be a win if we do not need this library anymore.

Reply:
I will try and remove it